playwright/api/basics.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  APIResponse,
+  request,
+} from "@playwright/test";
+
+const URL = process.env.API_URI || "";
+interface DATA {
```

Comment:
Types and interfaces should be written in pascal case
```suggestion
interface Data {
```